pkg/assembler/grammar.go

```diff
@@ -8,10 +8,10 @@ import (
 // Grammar and AST
 
 type CasmProgram struct {
-	Instructions []Instruction `@@*`
+	Ast []AstNode `@@*`
```

Comment:
`Ast` is a very generic name. Instructions, InstructionList, are more descriptive